README.md

```diff
@@ -10,4 +10,29 @@ Inspired by Webpack's [side-effects example].
 # Gettings started
 
 * Run `npm ci && git restore node_modules` to install all dependencies & restore the test node_modules that the repository includes.
-* Run `npm start` to build with webpack build & open the bundle analyzer.
+* Run `npm start` to build with Webpack build & open the bundle analyzer.
+
+# Notes
+
+Webpack can use treeshaking only when the final source files used for bundling are using the es6 import syntax.
```

Comment:
Maybe we can also mention that it should work similarly for other bundlers, but we haven't tested it, so we will refer to webpack here.

Reply:
Added that on top :+1: